api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentCalculator.java

```diff
@@ -37,18 +34,22 @@ public PaymentCalculation calculatePaymentCycleAmountInPence(PaymentCycleVoucher
         int maximumAllowedCardBalanceInPence = firstWeekEntitlementInPence * maximumBalancePeriod;
 
         if (isCardBalanceTooHigh(cardBalanceInPence, maximumAllowedCardBalanceInPence)) {
-            return NO_PAYMENT_CALCULATION;
+            return aBalanceTooHighPaymentCalculation(cardBalanceInPence);
         }
         int paymentCycleTotalEntitlementInPence = entitlement.getTotalVoucherValueInPence();
         if (fullPaymentKeepsBalanceWithinThreshold(cardBalanceInPence, maximumAllowedCardBalanceInPence, paymentCycleTotalEntitlementInPence)) {
             return PaymentCalculation.builder()
```

Comment:
Can we use the static constructors for these two constructions of `PaymentCalculation`?

Reply:
For the first one yes, I wouldn't for the second because it requires taking in two ints with no obvious parameter order. Would be very easy to get the two mixed up.